src/test/scala/uk/gov/hmrc/auth/core/AuthConnectorSpec.scala

```diff
@@ -64,7 +64,11 @@ class AuthConnectorSpec extends WordSpec with ScalaFutures {
       override val serviceUrl: String = "/some-service"
     }
 
-    def exceptionHeaders(value: String) = Map(AuthenticateHeaderParser.WWW_AUTHENTICATE -> s"""MDTP detail="$value"""")
+    def exceptionHeaders(value: String,enrolment:Option[String] = None) =
+      Map(
+        AuthenticateHeaderParser.WWW_AUTHENTICATE -> s"""MDTP detail="$value"""",
+        enrolment.map(e => AuthenticateHeaderParser.ENROLMENT -> e).getOrElse("" -> "")
```

Comment:
Instead of returning an empty field .getOrElse("" -> "") can we just not include the enrolment in the map?